docs/monster-ui/markdownEditor().md

```diff
@@ -0,0 +1,41 @@
+title: markdownEditor()
+
+# monster.ui.markdownEditor()
+The `monster.ui.markdownEditor()` function convert a simple textarea into a markdown editor.
+
+![](images/markdownEditor-preview.png)
+
+## Syntax
+```javascript
+monster.ui.markdownEditor(target[, options]);
+```
+
+### Parameters
+
+Key | Description | Type | Default | Required
+:-: | --- | :-: | :-: | :-:
+`target` | A jQuery object that will be converted to markdow editor. | `jQuery` | | `true`
```

Comment:
You should avoid writing in the future tense when describing parameters in general (as it makes it unnecessarily verbose).
```suggestion
`target` | A jQuery object to be converted into a markdown editor. | `jQuery` | | `true`
```